sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/tests/Azure.Monitor.OpenTelemetry.AspNetCore.Integration.Tests/DistroWebAppLiveTests.cs

```diff
@@ -181,31 +182,27 @@ public async Task VerifySendingToTwoResources_UsingExporter()
 
         [RecordedTest]
         [SyncOnly] // This test cannot run concurrently with another test because OTel instruments the process and will cause side effects.
-        [Ignore("TODO: Repurpose this test to validate the error.")]
```

Comment:
@rajkumar-rangaraj I'm planning to follow up with additional tests in the Tests project.
This project only hosts our Live tests. I want to keep only a few scenarios in here because it can be challenging to maintain these.